examples/qm9/qm9_dataset_exploration.ipynb

```diff
@@ -0,0 +1,558 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## QM9 Dataset exploration"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook is executed on 16, Jan, 2018. \n",
+    "\n",
+    "The tox21 dataset behavior or `chainer_chemistry` library behavior might change in the future."
```

Comment:
How about "The behavior of tox21 dataset in `chainer_chemistry` might change in the future."?